website/project/views/node.py

```diff
@@ -779,6 +779,8 @@ def _view_project(node, auth, primary=False):
                 'doi': node.get_identifier_value('doi'),
                 'ark': node.get_identifier_value('ark'),
             },
+            'institution': node.primary_institution.name if node.primary_institution else None,
```

Comment:
Minor:

> Prefer namespaced representations to arbitrary prefixes in response data. 

https://cosdev.readthedocs.org/en/latest/osf/guidelines.html#responses

So better to have: 

```json
{
  "institution": {
     "name": ...,
     "logo": ... 
 }
}
```